src/Models/Transfer.php

```diff
@@ -79,32 +80,32 @@ public function getTable(): string
     /**
      * @return BelongsTo<Wallet, self>
      */
-    public function from(): BelongsTo
+    public function from(): MorphTo
     {
-        return $this->belongsTo(Wallet::class, 'from_id');
+        return $this->morphTo();
     }
 
     /**
      * @return BelongsTo<Wallet, self>
      */
-    public function to(): BelongsTo
+    public function to(): MorphTo
     {
-        return $this->belongsTo(Wallet::class, 'to_id');
+        return $this->morphTo();
     }
 
     /**
      * @return BelongsTo<Transaction, self>
      */
     public function deposit(): BelongsTo
     {
-        return $this->belongsTo(Transaction::class, 'deposit_id');
+        return $this->belongsTo(config('wallet.transaction.model', Transaction::class), 'deposit_id');
     }
 
     /**
      * @return BelongsTo<Transaction, self>
      */
     public function withdraw(): BelongsTo
     {
-        return $this->belongsTo(Transaction::class, 'withdraw_id');
+        return $this->belongsTo(config('wallet.transaction.model', Transaction::class), 'withdraw_id');
```

Comment:
not relevant. everything works without problems.

detail: https://github.com/bavix/laravel-wallet/pull/858#issuecomment-1906640926